spyder/utils/palette.py

```diff
@@ -136,6 +139,9 @@ class SpyderPaletteLight:
     SPYDER_LOGO_WEB = Logos.B40
     SPYDER_LOGO_SNAKE = Logos.B50
 
+    # Colors for disabled state toolbar buttons
+    COLOR_DISABLED = ICON_1, OPACITY_DISABLED
```

Comment:
The same applies here.

Reply:
I double checked and these also match the values discussed in [ux-improvements #42](https://github.com/spyder-ide/ux-improvements/issues/42).

Reply:
Sorry @isabela-pf, I should have been clearer. I meant this line has the same problem of `OPACITY_DISABLED` not being prefixed by `Opacity`.

Reply:
No no no, I'm sorry I didn't catch it! Fixed now.